tests/core/pyspec/eth2spec/test/helpers/attestations.py

```diff
@@ -323,7 +323,8 @@ def temp_participants_filter(comm):
         next_slot(spec, state)
 
     assert state.slot == next_epoch_start_slot + spec.MIN_ATTESTATION_INCLUSION_DELAY
-    assert len(state.previous_epoch_attestations) == len(attestations)
+    if spec.fork != LIGHTCLIENT_PATCH:
```

Comment:
I would do `if spec.fork == PHASE0` or `if spec.fork in forks_list` rather than != LIGHTCLIENT.

Given that the accounting reform will carry forward to all forks starting at  light-client